functions/Skip-DbcCheck.ps1

```diff
@@ -5,22 +5,47 @@ Function Skip-DbcCheck {
         [object[]]$DbcchecksResult        
     )
     Process {
-        #Loading the Json Exclusion file
-        $Exclusion_File = (Get-PSFConfig dbapolicy.ExclusionsFilePath).Value
-        $Exclusions = Get-Content -Path $Exclusion_File -Raw | ConvertFrom-Json                
+        # Loading the Json Exclusion file
+        $Exclusion_File = (Get-PSFConfig dbachecks-ext.exclusion.filepath).Value
+        $Exclusions = Get-Content -Path $Exclusion_File -Raw | ConvertFrom-Json
 
-        #Removing the tests accordingly to the json exclusion file
+        # Collecting info regarding the number of the affected rows
+        $TestsSkipped = $DbcchecksResult | Foreach-Object {
```

Comment:
Redundant code, written twice. Can be calculated differently: Total number of TestResults minus all the results that were left.
```powershell
$before = ($DbccchecksResult.TestResult | Measure-Object).Count
$skippedResults = $DbccchecksResult | Foreach-Object { ... }
$after = ($skippedResults | Measure-Object).Count
$skipped = $before - $after
```
something like this